src/plugin/fieldViews/TextFieldView.ts

```diff
@@ -73,19 +79,25 @@ export class TextFieldView extends ProseMirrorFieldView {
       [keymap(keymapping)]
     );
 
-    if (enableMultiline) {
+    if (enableMultiline || isCode) {
       // We wait to ensure that the browser has applied the appropriate styles.
       setTimeout(() => {
         if (!this.innerEditorView) {
           return;
         }
-        const { lineHeight, paddingTop } = window.getComputedStyle(
-          this.innerEditorView.dom
-        );
+        if (enableMultiline) {
+          const { lineHeight, paddingTop } = window.getComputedStyle(
+            this.innerEditorView.dom
+          );
 
-        (this.innerEditorView.dom as HTMLDivElement).style.minHeight = `${
-          parseInt(lineHeight, 10) * rows + parseInt(paddingTop) * 2
-        }px`;
+          (this.innerEditorView.dom as HTMLDivElement).style.minHeight = `${
+            parseInt(lineHeight, 10) * rows + parseInt(paddingTop) * 2
+          }px`;
+        }
+        if (isCode) {
```

Comment:
We shouldn't have to wait a tick to apply the code styles – this is necessary in the case of `minHeight` because we must measure the element height to calculate the value.

As a result, it might be best to move the monospace styling out of this timeout, just to avoid confusion as to its purpose.

Reply:
Good point, I'm on it.